### ISSUE TITLE
Iris hyperemia

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10983]
Preferred term label:
Iris hyperemia

Synonyms
Iris hyperaemia, Increased iris vascularity, Congested iris vessels

Definition (free text, please give PubMed ID)
A condition characterized by increased blood flow and congestion of the iris vasculature, leading to a reddish discoloration of the iris. Iris hyperemia is commonly associated with anterior uveitis, angle-closure glaucoma, or ocular inflammation. 

Parent term (use hpo.jax.org/app)
Abnormality of the iris vasculature (HP:0007905)

Diseases characterized by this term
Anterior uveitis, Acute angle-closure glaucoma

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01 
Adding new term: 
- ID: HP:6001237
- label: Iris hyperemia
- parents: [Abnormal iris vasculature (HP:0007905)](https://hpo.jax.org/app/browse/term/HP:0007905)
- pmid: 
- def: A condition characterized by increased blood flow and congestion of the iris vasculature, leading to a reddish discoloration of the iris. Iris hyperemia is commonly associated with anterior uveitis, angle-closure glaucoma, or ocular inflammation.
- comment: 
- orcid: orcid:0000-0001-9042-0508
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10983
- synonyms: Congested iris vessels(NONE)|Increased iris vascularity(NONE)|Iris hyperaemia,(NONE)
